tests/filecheck/transforms/convert-arith-to-varith.mlir

```diff
@@ -69,6 +69,46 @@ func.func @test_mulf() {
     // CHECK-NEXT:   %a, %b, %c = "test.op"() : () -> (f32, f32, f32)
     // CHECK-NEXT:   %0, %1, %2 = "test.op"() : () -> (f32, f32, f32)
     // CHECK-NEXT:   %x2 = arith.mulf %0, %1 : f32
-    // CHECK-NEXT:   %r = varith.mul %c, %a, %b, %2, %0, %1 : f32
+    // CHECK-NEXT:   %r = varith.mul %a, %b, %c, %0, %1, %2 : f32
     // CHECK-NEXT:   "test.op"(%r, %x2) : (f32, f32) -> ()
 }
+
+func.func @test() {
+    %0, %1, %2, %3, %4, %5 = "test.op"() : () -> (tensor<8xf32>, tensor<8xf32>, tensor<8xf32>, tensor<8xf32>, tensor<8xf32>, tensor<8xf32>)
+    %6 = arith.constant dense<1.234500e-01> : tensor<8xf32>
+    %a = arith.addf %5, %4 : tensor<8xf32>
+    %b = arith.addf %a, %3 : tensor<8xf32>
+    %c = arith.addf %b, %2 : tensor<8xf32>
+    %d = arith.addf %c, %1 : tensor<8xf32>
+    %e = arith.addf %d, %0 : tensor<8xf32>
+    %12 = arith.mulf %e, %6 : tensor<8xf32>
```

Comment:
Why does this example use tensor types, instead of just f32 as above. The tensor type does not seem to by use anything. If it does, maybe point this out in a comment?

Reply:
done as suggested